simphony/io/h5_cuds.py

```diff
@@ -152,6 +155,47 @@ def add_mesh(self, mesh):
         self._file.flush()
         return m
 
+    def add_lattice(self, lattice, record=None):
+        """Add lattice to the file.
+
+        Parameters
+        ----------
+        lattice : Lattice or MetaLattice
+            lattice to be added or in the case of MetaLattice, a new
+            FileLattice is created.
+        record : tables.IsDescription, optional
+            CUBA-datatypes to be copied from lattice to FileLattice.
+            If 'None' then copy everything to a NoUIDRecord-table.
+
+        Returns
+        ----------
+        FileLattice
+            The lattice newly added to the file.  See
+            get_lattice for more information.
+
+        """
+        if lattice.name in self._file.root.lattice:
+            raise ValueError(
+                'Lattice \'{n}\` already exists'.format(n=lattice.name))
+        filelat = FileLattice(self, lattice.name, lattice.type,
+                              lattice.base_vect, lattice.size,
+                              lattice.origin, record)
+
+        # copy the contents of the lattice to the file
+        if isinstance(lattice, Lattice):
+            if record is None:
+                for node in lattice.iter_nodes():
+                    filelat.update_node(node)
+            else:
+                keywords = record.columns['data']._v_names
+                ckeys = list({eval(CUBA.__name__+'.'+k.upper())
```

Comment:
'eval' is not needed anymore here, since add_lattice only creates full tables, so I removed it. I'll try to find another way of selecting the CUBA-keys in the case that only some of them is saved into a file.